File: test_api.py
import requests

base_url = "https://ifo1usk4zzs6kf3w1axrg1y9.207.180.251.156.sslip.io"

def test_endpoint(path):
    url = f"{base_url}{path}"
    try:
        response = requests.get(url)
        print(f"{path}: {response.status_code} - {response.text[:100]}")
    except Exception as e:
        print(f"{path}: ERROR - {str(e)}")

print("Testing remote API endpoints:")
test_endpoint("/docs")
test_endpoint("/spaces")
test_endpoint("/spaces/my")
test_endpoint("/chat/conversations")

File: bin/test_api.dart
import 'package:dio/dio.dart';

void main() async {
  final dio = Dio(BaseOptions(baseUrl: 'https://ifo1usk4zzs6kf3w1axrg1y9.207.180.251.156.sslip.io'));
  
  try {
    final res = await dio.get('/spaces/my');
    print('SUCCESS: \${res.statusCode}');
  } on DioException catch (e) {
    print('ERROR: \${e.response?.statusCode} - \${e.message}');
  }
}
